src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/resource/AddElementDialog.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import { Button } from "@madie/madie-design-system/dist/react";
+import { Dialog, IconButton, DialogActions, Divider } from "@mui/material";
```

Comment:
Any reason why we are using Dialog from MUI instead of Design systems ?

Reply:
We may have to double check the Contrast with Micheal, Text seems hard to read once an attribute is selected